examples/00_tutorials/plot_single_subject_single_run.py

```diff
@@ -6,32 +6,13 @@
 :term:`fMRI` signal during periods of auditory stimulation
 versus periods of rest.
 
-The analyse described here is performed in the native space, directly on the
```

Comment:
remove dataset description as it can be directly printed from the dataset fetcher: see below

Reply:
- see #4218